blueprints/ember-bootstrap/index.js

```diff
@@ -34,7 +34,8 @@ module.exports = {
   },
 
   beforeInstall(option) {
-    let bootstrapVersion = parseInt(option.bootstrapVersion) || 3;
+    let configuredBootstrapVersion = this.retrieveBootstrapVersion();
+    let bootstrapVersion = parseInt(option.bootstrapVersion) || configuredBootstrapVersion || 3;
```

Comment:
You could inline `this.retrieveBootstrapVersion()` here instead of assigning it first to a variable, so it gets lazily evaluated only when needed. When we have `option.bootstrapVersion` set, we don't need to open and parse that file.